spec/bin/rollbar_runner_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+require 'rails/rollbar_runner'
+
+DUMMYAPP_DIR = File.expand_path('../../dummyapp', __FILE__)
+
+def with_dummyapp_dir(&block)
+  FileUtils.cd(DUMMYAPP_DIR) do
+    yield
+  end
+end
+
+describe Rails::RollbarRunner do
+  context 'with file contains define method in top level' do
+    # refs: https://github.com/rollbar/rollbar-gem/issues/273
+
+    before do
+      with_dummyapp_dir do
+        File.open('./lib/test_ruunner.rb', 'w') do |file|
```

Comment:
It'd be nice having this file written in the dummyapp instead of write it for every test.